src/app/i18n.ts

```diff
@@ -100,5 +89,39 @@ export function initi18n(): Promise<unknown> {
         }
       }
     );
+    if (DIM_LANG_INFOS[lang]?.pluralOveride) {
+      i18next.services.pluralResolver.addRule(lang, i18next.services.pluralResolver.getRule('en'));
+    }
   });
 }
+
+function getLanguagePath(language: string) {
```

Comment:
this function serves no purpose except to do a dictionary lookup, i'm not sure why it needs to be a function instead of just an object

if the goal is to make `loadPath` less messy, the languagePath object should be moved out to top level. it serves no purpose to create the object inside a function

Reply:
there was more in the function before I committed as I left the office...